src/indicator/algorithm/calculator/elderRay.js

```diff
@@ -0,0 +1,73 @@
+import d3 from 'd3';
+import exponentialMovingAverage from './exponentialMovingAverage';
+import {identity} from '../../../util/fn';
+import {rebind} from '../../../util/rebind';
+
+export default function() {
+
+    var value = identity;
+
+    var highValue = function(d, i) { return d.high; },
+        lowValue = function(d, i) { return d.low; };
+
+    var ema = exponentialMovingAverage()
+        .windowSize(13);
+
+    var elderRay = function(data) {
+
+        ema.value(value);
+        var x = ema(data);
+
+        var elderRay = d3.zip(data, x)
+            .map(function(d) {
```

Comment:
There's a bit of a problem with this calculator, it doesn't accomodate the fact that the ema will have 'leading' undefined values. Look at where undefinedInputAdapter is used, in MACD and Envelope, this should hopefully allow you to ignore the undefined values in your calculator.